cmd/crc/cmd/delete.go

```diff
@@ -24,31 +24,32 @@ var deleteCmd = &cobra.Command{
 	Short: "Delete the OpenShift cluster",
 	Long:  "Delete the OpenShift cluster",
 	Run: func(cmd *cobra.Command, args []string) {
-		runDelete(args)
+		if err := runDelete(args); err != nil {
+			errors.ExitWithMessage(1, err.Error())
+		}
 	},
 }
 
 var clearCache bool
 
-func runDelete(arguments []string) {
-	deleteConfig := machine.DeleteConfig{
-		Name: constants.DefaultName,
-	}
-
+func runDelete(arguments []string) error {
 	if clearCache {
 		deleteCache()
 	}
 
-	exitIfMachineMissing(deleteConfig.Name)
```

Comment:
we left the option for a name because some people want the multi instance option. at the moment we do not prevent people using a derived command that used the Name tto deal with this. 

in our own commands we do not use this.

Reply:
OK, I will put it back.